### PR TITLE
Fix the build.

### PR DESCRIPTION
**Pull request by girum11**

### CODE REVIEW

**Comment by girum11** [comment]
LGTM

File: tools/config/seed.config.ts [hunk 1]
```diff
@@ -321,7 +321,7 @@ export class SeedConfig {
       '@angular/http': `node_modules/@angular/http/bundles/http.umd.js`,
       '@angular/platform-browser': `node_modules/@angular/platform-browser/bundles/platform-browser.umd.js`,
       '@angular/platform-browser-dynamic': `node_modules/@angular/platform-browser-dynamic/bundles/platform-browser-dynamic.umd.js`,
-      '@angular/router': `node_modules/@angular/router/index.js`,
+      '@angular/router': 'node_modules/@angular/router/bundles/router.umd.js',
```
Comment (nati22) [inline tools/config/seed.config.ts]: What's happening here?

**Comment by nati22** [comment]
LGTM

File: tools/config/seed.config.ts (hunk 1 above)
Comment (girum11) [inline tools/config/seed.config.ts]: Angular 2.2 changed the bundle path of @angular/router  =\

**Pull request merged**